ui/panels-plugin/src/plugins/markdown/MarkdownPanel.tsx

```diff
@@ -11,17 +11,58 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import { useMemo } from 'react';
 import * as DOMPurify from 'dompurify';
 import { marked } from 'marked';
-import { Box } from '@mui/material';
+import { Box, Theme } from '@mui/material';
 import { PanelProps } from '@perses-dev/plugin-system';
 import { MarkdownPanelOptions } from './markdown-panel-model';
 
 export type MarkdownPanelProps = PanelProps<MarkdownPanelOptions>;
 
-// Overall, this panel does:
-// 1. text --> html (with support for original markdown and GFM)
-// 2. sanitize the html to prevent XSS attacks
+function createMarkdownPanelStyles(theme: Theme) {
```

Comment:
Oh cool, I've had to do styles this way a few times to override lib styles, I should have remembered to suggest it too!

Reply:
Yeah, I think my mental blocker was getting the insight that I can apply styling in all the ways people normally do with `mui`, it doesn't need to be different because I'm writing code that's part of library that will be used by other projects. 

(And realizing that that assumption works because we're requiring that people install `mui` to use our code.)